scripts/auth-open.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+import json
+import os
+import base64
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+#
+# From : https://github.com/dcos/dcos/blob/master/test_util/helpers.py#L35
```

Comment:
We're only using the constant and from it's current location I couldn't simple import the respective data... Do you think it makes sense to restructure the `helpers.py` so that we have an `auth_helper.py` which provides the respective functionality for both the system test  driver as well as the integration tests?